### ISSUE TITLE
SimGetImage looks very dark

### ISSUE THREAD

**Issue by xuhao1** [microsoft/AirSim#1144]
Hi everyone,
I am using SimGetImage to grab image from Unreal Enviroment , but the image looks really dark after I set up all the parameter (like AutoExposureSpeed and so on.)
And also I tried PNG and uncompressed image, nothing changed
<img width="1865" alt="capture" src="https://user-images.githubusercontent.com/5087930/41139805-a0e4b45e-6b1c-11e8-86ef-6fb579d64500.PNG">

and also two image in below should be the same
<img width="1343" alt="capture2" src="https://user-images.githubusercontent.com/5087930/41140135-47124066-6b1e-11e8-81b7-2c4b29eae340.PNG">
But the one display by python is darker then another

**Comment by kuanting** [comment]
I encountered the same issue in several environments, is it something related to Unreal lighting condition?

The screen I saw in Unreal engine
![vivid 64-bit pcd3d_sm5 6_12_2018 4_45_33 pm](https://user-images.githubusercontent.com/2151278/41279927-4f63f1e8-6e60-11e8-8d4e-fc5a837a03f5.png)

The screenshot I got using AirSim API
![screenshot_0](https://user-images.githubusercontent.com/2151278/41279984-74ba5c5c-6e60-11e8-8eb4-4deb0c4dfb52.png)

**Comment by DavidLSmyth** [comment]
I found that changing the bp_sky_sphere sun brightness to around 80 made the brightness of the recorded images correspond more closely to the viewport images. Changing the auto-exposure min and max brightness in post-process volume(s) may also be worth looking at from my previous experimentation

**Comment by sytelus** [comment]
You might also want to adjust TargetGamma:

```
"CameraDefaults": {
    "CaptureSettings": [
      {
        "ImageType": 0,
        "TargetGamma": 1.5
      }
    ]
}
```

Also see [note on new settings](https://github.com/Microsoft/AirSim/blob/master/docs/upgrade_settings.md).

**Comment by mjcorriere** [comment]
@sytelus Can sensible defaults please be provided such that the output images match the in game images? It is frustrating that additional work needs to be done to capture the exact images that are being shown to the user on screen.

**Comment by AloshkaD** [comment]
@kuanting do you mind if I ask you where did you get the drone geometry you have and if you mind sharing the physics script as well. Appreciated!

**Comment by zimmy87** [comment]
This looks like it was fixed by #1834, so closing
